components/Article/PayNote.js

```diff
@@ -132,44 +154,49 @@ const getPayNoteVariation = ({
     }
   }
   return isEligibleForTrial && (inNativeIOSApp || trial)
-    ? getTryVariation(seed, { query })
-    : getBuyVariation(seed, { query, isSeries })
+    ? getTryNote(seed, {
+        query,
+        customNotes: getTryNotes(customPayNotes)
+      })
+    : getBuyNote(seed, {
+        isSeries,
+        customNotes: getBuyNotes(customPayNotes)
+      })
 }
 
 const Translation = compose(
   withT,
   graphql(memberShipQuery)
-)(({ t, data: { membershipStats }, baseKey, position, element }) => {
-  const tKey = [baseKey, position, element].filter(el => el).join('/')
+)(({ t, data: { membershipStats }, payNote, position, element }) => {
+  const key = payNote.key
+    ? [payNote.key, position, element].filter(el => el).join('/')
+    : `${position}${capitalize(element) || 'Body'}`
   const count = countFormat((membershipStats && membershipStats.count) || 20000)
+  const missingKey = payNote.payNote ? payNote.payNote[key] : ''
```

Comment:
```suggestion
  const customText = payNote.custom && payNote.custom[key]
```

Reply:
We need the missingKey set to empty string also when using one of the preset paynotes. Otherwise it shows the default missing key TK(${key}) when the title isn't specified (which happens for a few paynotes)